xpdacq/beamtime.py

```diff
@@ -365,17 +365,8 @@ def __init__(self,sample, scanplan):
         self.type = 'sc'
         self.sp = scanplan
         self.sa = sample
-        self.md = self.sp.md
+        self.md = dict(self.sp.md) # create a new dict copy.
```

Comment:
still not sure about this.  Does it pass all the tests?

I still think it would be better to leave the ``sp`` and ``sa`` metadata in ``self.sp.md`` and so on, then just ``self.md = {}``

Then right before RE we can create a 
```
metadatadict = sc.md
metadatadict.update(sc.sp.md)
metadatadict.update(sc.sa.md)
xpdRE(..., metadatadict)
```
sthg like that.  We can do away with ``Union``, which I wrote but I don't think I like it now.